pkg/collector/sharedlibrary/helpers.go

```diff
@@ -0,0 +1,67 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// the content of this file is a part of pkg/collector/python/helpers.go
```

Comment:
Same comment as previously, we should be able to reuse the functions defined in the other package ?